templates/landing.html

```diff
@@ -41,9 +41,9 @@
         <div class="explore-button-container">
             <button class="button" onclick="window.location.href='/messages'">Messages <i class="fa fa-comments"></i></button>
         </div>
-      <div class="explore-button-container">
-                <button class="button" onclick="window.location.href='/profile'">Profile <i class="fa fa-user"></i></button>
-            </div>
+        <div class="explore-button-container">
```

Comment:
Let's revert this since we want to just keep one 'Profile" button which is in the landing page.